packages/input-text/README.md

```diff
@@ -0,0 +1,15 @@
+
+
+# Farmblocks Text Input
+
+A form field to input simple text
+
+## Installation
+
+```
+npm install @crave/farmblocks-input-text
+```
+
+## License
+
+MIT
```

Comment:
todo: write readme

Reply:
this document will be real useful here, to describe which properties and apis we mirror from html and which ones differs or are unimplemented, will write this